### PR TITLE
master-qa-52718

### PR DESCRIPTION
**Pull request by zhangyan0701**
https://issues.liferay.com/browse/LRQA-52718

### CODE REVIEW

**Comment by zhangyan0701** [comment]
ci:test:sf

**Comment by zhangyan0701** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase [hunk 1]
```diff
@@ -2029,6 +2029,12 @@ definition {
 
 		Form.editDescription();
 
+		var browserType = PropsUtil.get("browser.type");
+
+		if (("${browserType}" == "internetexplorer") && (IsElementPresent(locator1 = "Form#SIDEBAR_CLOSE"))) {
+			Click(locator1 = "Form#SIDEBAR_CLOSE");
+		}
+
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase]: Is this a consistent issue on CI? Are you able to reproduce locally?

**Comment by john-co** [comment]
Pushed to BChan
master: https://github.com/brianchandotcom/liferay-portal/pull/81522
7.2.x: https://github.com/brianchandotcom/liferay-portal-ee/pull/27868

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase (hunk 1 above)
Comment (zhangyan0701) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase]: I am not able to reproduce on local. On CI, it was consistent before the draganddrop issue happened, but not the draganddrop issue disappeared somehow, and the error changed back to this one.

**Pull request closed without merging**